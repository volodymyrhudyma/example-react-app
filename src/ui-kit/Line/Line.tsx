import React from 'react';
import { Wrapper } from './styled/Line.styled';

interface ILineProps {
  width: string;
  height: string;
}

const Line = (props: ILineProps) => <Wrapper {...props} />;

Line.defaultProps = {
  width: '100%',
  height: '4px',
};

export default Line;
