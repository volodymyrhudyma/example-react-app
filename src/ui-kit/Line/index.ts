import Component from './Line';

export default Component;
