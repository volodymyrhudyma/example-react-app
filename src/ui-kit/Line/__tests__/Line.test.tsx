import React from 'react';
import { shallow } from 'enzyme';
import Line from '../Line';

describe('Line component', () => {
  it('should render correctly', () => {
    const component = shallow(<Line />);
    expect(component).toMatchSnapshot();
  });
});
