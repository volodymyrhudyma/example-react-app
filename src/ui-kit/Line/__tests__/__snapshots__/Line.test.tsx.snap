// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line component should render correctly 1`] = `
<styled.div
  height="4px"
  width="100%"
/>
`;
